policy/policy.go

```diff
@@ -30,6 +30,31 @@ type List struct {
 	Policies      map[string]TLSPolicy `json:"policies"`
 }
 
```

Comment:
note: comment above TLSPolicy.Mode that it can only be one of `none` `enforce` `testing`